src/test/java/com/org/jenkins/custom/jenkins/distribution/service/PluginControllerTest.java

```diff
@@ -36,7 +36,7 @@ public void return_status_200() throws Exception {
         String uri = "/api/plugin/getPluginList";
         MvcResult mvcResult = mvc.perform(MockMvcRequestBuilders.get(uri).accept(MediaType.APPLICATION_JSON_VALUE)).andReturn();
         int status = mvcResult.getResponse().getStatus();
-        assertThat(new JSONObject(dummyUpdateBody).toString(), CoreMatchers.containsString(mvcResult.getResponse().getContentAsString()));
+        assertThat(mvcResult.getResponse().getContentAsString(), CoreMatchers.containsString(new JSONObject(dummyUpdateBody).toString()));
```

Comment:
Check if the response contains the update center dummy body because if the map is removed it returns null so this check satisfies our concern that the test will fail if the map is removed

Reply:
Thank you!